core/src/main/java/com/yahoo/oak/HashChunk.java

```diff
@@ -274,7 +274,7 @@ boolean finalizeDeletion(ThreadContext ctx) {
 
 
     /********************************************************************************************/
-    /*------------------------- Methods that are used for rebalance  ---------------------------*/
+    /*------------------------- Methods that are used for rebalanced  ---------------------------*/
```

Comment:
This is indeed "rebalance", a noun that we invented, although not understood by spell checker :)